src/containers/Elements/NodeForm.js

```diff
@@ -51,50 +61,70 @@ class NodeForm extends Component {
     }, this.submit);
   };
 
+  isLarge = () => window.matchMedia('screen and (min-device-aspect-ratio: 16/9)').matches;
+
   render() {
     const {
-      title,
-      fields,
-      name,
       addAnother,
+      autoPopulate,
+      fields,
       initialValues,
+      name,
+      title,
     } = this.props;
 
+    const modalClassNames = cx({ 'modal--fullscreen': !this.isLarge() });
+
+    const props = {};
```

Comment:
Same as `extraProps` in wizard from.

Would probably take away the need to specify `autoPopulate` in propTypes.